model/scikit/tests/test_scikit.py

```diff
@@ -129,9 +129,11 @@ def tearDownClass(cls):
 
     async def test_02_predict(self):
         async with self.sources as sources, self.model as model:
+            target = model.config.predict.NAME
             async with sources() as sctx, model() as mctx:
                 async for repo in mctx.predict(sctx.repos()):
-                    prediction = repo.prediction().value
+                    # prediction = repo.prediction(target).value
+                    prediction = repo.prediction(target)["value"]
```

Comment:
This shouldn't have changed to `['value']` we need to make sure we're creating `RepoPrediction` instances.